src/components/Combobox/__tests__/combobox-tests.jest.js

```diff
@@ -16,6 +16,15 @@ describe("Combobox tests", () => {
     { value: "yellow", label: "Yellow" }
   ];
 
+  it("should call item on click callback func when onClick", () => {
+    const onClickMock = () => console.log("%%%");
```

Comment:
I know, this test is important but not working... can we talk about it after the review?